components/browser/session/src/main/java/mozilla/components/browser/session/storage/DefaultSessionStorage.kt

```diff
@@ -110,12 +110,12 @@ class DefaultSessionStorage(
             json.put(VERSION_KEY, VERSION)
             json.put(SELECTED_SESSION_KEY, sessionManager.selectedSession.id)
 
-            sessionManager.sessions.forEach({ session ->
+            sessionManager.sessions.filter { it.customTabConfig == null }.forEach { session ->
```

Comment:
I wonder if this should be the default for `sessions` (return the filtered list). A lot of things are going to need the filtered list (e.g. tabs tray) and so far nothing in Focus required *all* sessions. We only look up specific sessions by ID for custom tabs.

Reply:
OK, I will update.